app/views/pruebas/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class='columns'>
+    <div class='column'></div>
```

Comment:
Do you need this empty column?
If space is wat you want, bulma has a `offset` class that can help you. Avoid using empty container.